src/main/front/src/components/Main.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import '../App.css';
+import Board from './board/Board';
+import styled from 'styled-components';
+import Sidebar from './Sidebar';
+
+const MainWrapper = styled.div`
```

Comment:
Probably something there should be changed to avoid white spaces on browser resize